### DISCUSSION TITLE
Discord phone verification

### DISCUSSION THREAD

**Discussion by alepersson** [chromiecraft/chromiecraft#3223] [General]
This made me unable to join your Discord server. Phone verification is a heavy-handed, privacy infringing method of validating users. Discord, a messaging service, has no business storing users' phone numbers. I understand that this is a security option preset you may have chosen because it is convenient, but I implore you to consider other options of verifying your user base.

**Comment by 55Honey** [comment]
Hello aleHansson,
Thanks for reaching out.

It's not in us to decide how discord verifies their accounts and we do not get in touch with phone-numbers either.
However, the only way to enforce a somewhat strong burden against fake accounts, is discords phone verification. The remaining options can be easily bypassed by bot or throwaway accounts.

There are no plans to change this requirement.

**Reply by ypoptos** [reply]
I also don't have discord because of the phone verification. I refuse to give my phone without reason. You demand my phone for discord access but you are ok to get my money without my phone (I paid twice but because of the discord demand I am not sure if I will do it again...)

**Reply by Nyeriah** [reply]
Hey @ypoptos 

We appreciate your willingness to help. However, as mentioned above, it's the only reliable feature Discord offers to validate accounts and somewhat deter people from registering fake accounts that do nothing but harm the community in the long run.

The validation process is entirely performed by Discord and we don't have any access to your phone numbers. Discord requires it during the verification process. We never ask for your phone number and we do not retain such data.

Regards.

**Reply by ypoptos** [reply]
I am using Discord many years with many servers and I never gave my phone for validation. This is a demand from you and not from Discord and I find it really bad and strange. Even if you don't see my phone someone else see it and stores it. And if the phone verification will be also a must demand from Discord I will stop using it. What the hell, everyone asks and stores our email and our phone number and someone else scamming us.

**Reply by Evelmike** [reply]
That's the rule we all follow. They are not going to change it for you. Please accept this or move on.

**Reply by WarcraftRefugee** [reply]
Completely agree. I have never seen this demanded just to read someone's Discord forums. No, you cannot have my cellphone#. nope

**Reply by WarcraftRefugee** [reply]
> Hey @ypoptos
> 
> We appreciate your willingness to help. However, as mentioned above, it's the only reliable feature Discord offers to validate accounts and somewhat deter people from registering fake accounts that do nothing but harm the community in the long run.
> 
> The validation process is entirely performed by Discord and we don't have any access to your phone numbers. Discord requires it during the verification process. We never ask for your phone number and we do not retain such data.
> 
> Regards.

This reply just made me completely turned off by Chromiecraft. You already have a TERRIBLE customer service/moderation reputation.  This is not a Discord issue; I am a member of over 25 Discord servers, 5 of them WoW private servers, and none have ever demanded my phone# just to read forums. I know you know this. And I know that YOU know.... we all know this. So your slimy reply to this thread has just killed my interest in this server. For a dying server, things like this do nothing to keep it alive.

**Reply by Evelmike** [reply]
So go play on one of those other 25 servers and stop wasting everyone's time here.

**Reply by WarcraftRefugee** [reply]
> So go play on one of those other 25 servers and stop wasting everyone's time here.

yep already done. You sound threatened by me - and the original poster's - statement of facts. lol typical Chromiecraft nonsense. Are you the trigger happy moderator who everyone complains about? Dont spend your life afraid of the truth.

**Reply by WarcraftRefugee** [reply]
> So go play on one of those other 25 servers and stop wasting everyone's time here.

But before I leave... consider that maybe replies like yours is one of the reasons this server is dying. Just consider it. have a good day
